packages/core/src/database/migrations/z1687287269-update-community.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+// eslint-disable-next-line no-undef
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        if (process.env.NODE_ENV === 'test') {
+            return;
+        }
+
+        await queryInterface.removeConstraint('ssi', 'ssi_communityPublicId_fkey');
+        await queryInterface.removeConstraint('notifiedbacker', 'notifiedbacker_communityId_fkey');
+        await queryInterface.removeConstraint('ubi_request_change_params', 'ubi_request_change_params_communityId_fkey');
+        
+        await queryInterface.removeColumn('community', 'publicId');
+        await queryInterface.removeColumn('community', 'descriptionEn');
+        await queryInterface.removeColumn('community', 'coverImage');
+        await queryInterface.removeColumn('community', 'coverMediaId');
+    },
+
+    down(queryInterface, Sequelize) {},
+};
```

Comment:
remember that we have [preboot](https://devcenter.heroku.com/articles/preboot) enabled. If you run this while the current backend is using it, it's going to break. One possible way is to allow it to run only after X timestamp (like, in 1 month).

Reply:
I forgot this detail...
I moved the migration to the "future_migrations" to run later, I think it's easier right?

Reply:
No relly, because we will forget I think. There are a few more there that need to be moved. But lets merge it now